### PR TITLE
Feat: added the slider component

### PR DESCRIPTION
**Pull request by pradipgharge**
added slider and refactored some code, updated readme.

### CODE REVIEW

File: pages/components/Badge/badge.css [hunk 1]
```diff
@@ -87,4 +64,31 @@
...
+    width: 100%;
+    height: auto;
+}
+
+.bd-count {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    position: absolute;
+    font-size: 12px;
```
Comment (mehulsatardekar) [inline pages/components/Badge/badge.css]: use rem property here

**Review by mehulsatardekar — APPROVED** [review]
good to merge the branch after changes

**Review by ashishkatiyar2404 — APPROVED** [review]
Appreciation: -  All good code has been followed, indentations, consistent variables name, class names also, understandable naming conventions, easy to read overall everything looks good.💯

File: pages/components/Badge/badge.css (hunk 1 above)
Comment (pradipgharge) [reply pages/components/Badge/badge.css]: Done

**Review by aamrapaliwandhre — APPROVED** [review]
code looks good to me...

**Review by Sahil25061999 — APPROVED** [review]
The slider component looks good, a ui suggestion would be, on hover use a lighter shade of your primary color, rest looks good.

**Comment by pradipgharge** [comment]
> The slider component looks good, a ui suggestion would be, on hover use a lighter shade of your primary color, rest looks good.

Done

**Review by eakansh125 — APPROVED** [review]
Could not find a syntactic issue. The slider looks neat and smooth to use.
Can be merged from my side.

**Pull request merged**